Formula/genders.rb

```diff
@@ -16,6 +16,8 @@ class Genders < Formula
   def install
     args = ["--prefix=#{prefix}"]
 
+    args << "--with-java-extensions=no"
```

Comment:
Since this isn't a conditional `configure` argument (unlike the one just below), you can merge it with the initialization of the `args` array from above like this:

```ruby
args = %W[
  --prefix=#{prefix}
  --with-java-extensions=no
]
```